docs/oauth.md

```diff
@@ -50,7 +50,21 @@ In your connectionBuilder.js file you need to use your DB sepcific logic to hand
       params["TOKEN"] = attr["ACCESSTOKEN"];
   }
 ```
+In your connectionResolver.tdr file, the following related OAuth attrs will be auto included so you do not need to define them:
+```ACCESSTOKEN, REFRESHTOKEN, access-token-issue-time, access-token-expires-in, CLIENTID, CLIENTSECRET, oauth-client, id-token(if any), instanceurl(if any)```
```

Comment:
I thought instanceurl was required and not auto-included? It works on desktop, but for server I had to add it manually to the required attributes list, otherwise I couldn't connect to a workbook or datasource published with that connection. (I also had to add a default-value to the instanceurl field in the dialog).

Reply:
Nope you should not need that, it will be auto-included if it exsits see  https://sourcegraph.prod.tableautools.com/teams/near/-/blob/modules/connectors/tabmixins/main/db/BaseConnectionResolver.cpp#L34:48.
You however need to define instanceUrl in your connection-feidls. https://gitlab.tableausoftware.com/connectivity/plugins/azure-dw/-/blob/master/src/script/connection-fields.xml#L17